src/Stitching/Stitching/Middleware/DelegateToRemoteSchemaMiddleware.cs

```diff
@@ -33,20 +33,22 @@ public async Task InvokeAsync(IMiddlewareContext context)
             if (context.Field.Directives.Contains(DirectiveNames.Delegate))
             {
                 NameString schemaName = default;
+                OperationType operationType = OperationType.Query;
                 IReadOnlyCollection<IError>? errors = null;
 
                 foreach (DelegateDirective delegateDirective in context.Field
                     .Directives[DirectiveNames.Delegate]
                     .Select(t => t.ToObject<DelegateDirective>()))
                 {
                     schemaName = delegateDirective.Schema;
+                    operationType = delegateDirective.Operation;
```

Comment:
the auto stitched root types should enable us to infer the default operation type in a better way. not sure yet how I get access to this information.